packs/src/heroes/akra-dragonborn-cleric.json

```diff
@@ -5295,6 +5211,40 @@
         "modifiedTime": 1661603486213,
         "lastModifiedBy": "dnd5ebuilder0000"
       }
+    },
+    {
+      "name": "Priest",
+      "type": "background",
+      "system": {
+        "description": {
+          "value": "<ul><li><strong>Skill Proficiencies:</strong> Insight, Religion</li><li><strong>Languages:</strong> Two of your choice</li><li><strong>Equipment:</strong> A holy symbol, 5 sticks of incense, prayer book, vestments, a set of common clothes, and a pouch containing 15 gp.</li></ul>",
+          "chat": "",
+          "unidentified": ""
+        },
+        "source": "",
+        "advancement": []
+      },
+      "img": "icons/sundries/documents/document-torn-diagram-tan.webp",
+      "effects": [],
+      "folder": null,
+      "sort": 0,
+      "ownership": {
+        "default": 0
+      },
+      "flags": {
+        "core": {
+          "sourceId": "Item.VY4Dc9uJPNVlvu1D"
```

Comment:
This is a little suspicious, it looks like a world item maybe? Since these backgrounds are custom, I probably wouldn't expect them to have a `sourceId` at all.